db/seeds.rb

```diff
@@ -1,7 +1,25 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+if User.where(name: "Test user").present?
+  puts "Skipping because user already exists"
+else
+  gds_organisation_id = "af07d5a5-df63-4ddc-9383-6a666845ebe9"
+  User.create!(
+    name: "Test user",
+    permissions: ["signin", "GDS Admin"],
```

Comment:
Are you confident these are the perms you need? I don't know Whitehall perms very well myself so not sure if GDS Admin supersedes ones like Editor and GDS Editor - @gpeng do you happen to know?

Reply:
We found using `gds_editor` didn't allow us to create organisations within the frontend so we needed it for manual testing at least

Reply:
I think `GDS Admin` is about the highest permission so unless it's the permissions we're testing, which seemingly it isn't, that's probably a good choice.